cli/utils.go

```diff
@@ -15,9 +15,9 @@ import (
 
 var (
 	// Limit query parameter.
-	Limit uint = 10
+	Limit uint64 = 10
```

Comment:
Why do we need `uint64` here instead of simple `uint`?

Reply:
I am using the [https://pkg.go.dev/strconv#ParseUint](ParseUint) function which returns a uint64. So I changed that to match the return type.